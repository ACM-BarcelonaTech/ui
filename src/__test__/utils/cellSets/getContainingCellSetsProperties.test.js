import getContainingCellSetsProperties from 'utils/cellSets/getContainingCellSetsProperties';
import mockCellSets from 'utils/tests/mockStores/cellSets';

describe('get cell class properties tests', () => {
  const { cellSets } = mockCellSets();
  it('returns information for a passed cellId', () => {
    const cellId = 3;
    const cellSetClassKeys = ['test'];
    const result = getContainingCellSetsProperties(cellId, cellSetClassKeys, cellSets);

    expect(result).toMatchSnapshot();
  });
});
