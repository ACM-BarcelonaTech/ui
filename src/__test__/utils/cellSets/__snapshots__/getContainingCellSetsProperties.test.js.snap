// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`get cell class properties tests returns information for a passed cellId 1`] = `
Object {
  "test": Array [
    Object {
      "cellIds": Set {
        1,
        2,
        3,
      },
      "name": "Test-1",
    },
  ],
}
`;
